### ISSUE TITLE
Readme updates needed for mid-term plans?

### ISSUE THREAD

**Issue by Splaktar** [PolymerElements/paper-elements#76]
### Description

The readme says the following since 11/2015:

```
Mid-term

paper-bottom-sheet - a material design bottom sheet this can easily be created with neon-animation (example)
paper date and time pickers - material design date and time pickers
paper steppers - material design steppers
paper cascading menu - the desktop cascading version of the material design dropdown menus
```
### Expected outcome

After 9 months, it would be expected that at least one mid-term component would have been implemented or moved to the short-term list.
### Actual outcome

But from what I hear, the paper-elements may be undergoing significant code changes 'soon'. There was also the announcement that work on new elements was going to be frozen to focus on the quality of existing elements.

So in 9 months, there has been no update to any of the short/mid/long term expectations in the README. I'm not complaining about these free components, but I think that it would reduce confusion if these expectations could be updated in 2016 :)
### Live Demo

https://github.com/PolymerElements/paper-elements/blob/master/README.md

**Comment by tjsavage** [comment]
Fair point, we'll update the roadmap shortly. Our focus on the paper elements has been on fixing bugs and improving performance. Over the last 9 months of maintaining the paper elements and also exploring other similar component products, we've found there's a very direct tradeoff between component quality and number of components available. So we've been extremely cautious to introduce new components, and focused primarily on bugfixes and quality improvements to the existing components.

We've also seen a lot of great work in the broader web components community around creating some additional material design components (e.g. [date picker](https://customelements.io/bendavis78/paper-date-picker/), [chip](https://customelements.io/bendavis78/paper-chip/), [data table](https://customelements.io/David-Mulder/paper-datatable/), etc.). We're also in the process of planning what the next version of the paper-\* and iron-\* elements might look like - again while continuing maintenance and making improvements to the existing set - and investigating ways we might decrease the maintenance burden. We'll update the roadmap to reflect this.

**Comment by Splaktar** [comment]
Sounds good. Thank you.
